source/agora/node/Ledger.d

```diff
@@ -49,6 +54,9 @@ public class Ledger
     /// UTXO set
     private UTXOSet utxo_set;
 
+    /// Callback when a transaction set is ready to be nominated
+    public NominatorDg nominateTxSet;
```

Comment:
Why not just make it a method and pass its address when needing a delegate ?

Reply:
This was accidentally public, it should be private. I'll fix it.

Reply:
I mean it could still be a method instead of a delegate no?